### DISCUSSION TITLE
Value mapping

### DISCUSSION THREAD

**Discussion by iBelow** [grafana/grafana#61483] [Dashboards, Panels & Visualizations]
hi, how i can use in value mapping current variable? 
for example
i want to change value 
from 
A: 10
to
a: ↓10 or ↑10
![image](https://user-images.githubusercontent.com/8112733/212322266-24249e64-9cd7-488b-8bff-4615ae21bb61.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
